src/lib/hardware/pci.lua

```diff
@@ -54,9 +54,12 @@ function path(pcidev) return "/sys/bus/pci/devices/"..pcidev end
 -- Return the name of the Lua module that implements support for this device.
 function which_driver (vendor, device)
    if vendor == '0x8086' then
-      if device == '0x10fb' then return 'apps.intel.intel10g' end -- Intel 82599
-      if device == '0x10d3' then return 'apps.intel.intel' end    -- Intel 82574L
-      if device == '0x105e' then return 'apps.intel.intel' end    -- Intel 82571
+      if device == '0x10fb' then return 'apps.intel.intel_app' end -- Intel 82599
```

Comment:
This change has broken the packetpblaster program, which checks for the driver `apps.intel.intel10g` explicitely.